src/redux/books/book.js

```diff
@@ -0,0 +1,60 @@
+export const ADD_BOOK = 'ADD_BOOK';
+export const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
[REQUIRED] Kindly follow the docs pattern in your action types value. It should be like this `bookstore/books/REMOVE_BOOK`
[REQUIRED] You should be exporting action container methods, not the action type variable